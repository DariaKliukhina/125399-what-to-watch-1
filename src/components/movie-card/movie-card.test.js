import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';

const mock = {
  film: {
    genre: [`Dramas`],
    title: `Macbeth`,
    picture: `picture.jpg`,
    preview: `video.mp4`
  }
};

it(`MovieCard renders correctly`, () => {
  const {film} = mock;

  const tree = renderer
    .create(<MovieCard
      film={film}
      onGenreClick={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

