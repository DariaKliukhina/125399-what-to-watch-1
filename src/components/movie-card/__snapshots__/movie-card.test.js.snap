// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <button
    className="small-movie-card__play-btn"
    onClick={[Function]}
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Macbeth"
      height="175"
      src="img/picture.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Macbeth
    </a>
  </h3>
</article>
`;
